Client/mods/deathmatch/logic/CPlayerMap.cpp

```diff
@@ -752,3 +807,209 @@ SString CPlayerMap::GetBoundKeyName(const SString& strCommand)
         return strCommand;
     return pCommandBind->boundKey->szKey;
 }
+bool CPlayerMap::SetCustomMapImage(const std::string& strTexturePath, uint uiSize, CResource* pResource)
+{
+    if (uiSize != 1024 && uiSize != 2048)
```

Comment:
How about using a namespace instead of uint?
`
enum class ECustomMapResolution
{
    Res_1024,
    Res_2048
}
`
Because `uint uiSize` argument takes just two possible values. This is a sign that something is wrong with the current implementation.